cosmos/constants.py

```diff
@@ -160,3 +160,7 @@ def _missing_value_(cls, value):  # type: ignore
 TESTABLE_DBT_RESOURCES = {DbtResourceType.MODEL, DbtResourceType.SOURCE, DbtResourceType.SNAPSHOT, DbtResourceType.SEED}
 
 DBT_COMPILE_TASK_ID = "dbt_compile"
+
+TELEMETRY_URL = "https://astronomer.gateway.scarf.sh/astronomer-cosmos/{telemetry_version}/{cosmos_version}/{airflow_version}/{python_version}/{platform_system}/{platform_machine}/{event_type}/{status}/{dag_hash}/{cosmos_task_count}"
+TELEMETRY_VERSION = "v1"
+TELEMETRY_TIMEOUT = 5.0
```

Comment:
Can we make this configurable? If not, can we set a lower value of 1-2 seconds? 5 seconds could be a bit large wait time and if it runs in the scheduler could cause some blocking experience

Reply:
For now I reduced to 1.0 - we can consider making it configurable in future, if needed